go-controller/pkg/clustermanager/userdefinednetwork/controller.go

```diff
@@ -240,6 +248,30 @@ func (c *Controller) SyncUserDefinedNetwork(udn *userdefinednetworkv1.UserDefine
 	return nad, nil
 }
 
+func (c *Controller) verifyNetAttachDefNotInUse(nad *netv1.NetworkAttachmentDefinition) error {
+	pods, err := c.podInformer.Lister().Pods(nad.Namespace).List(labels.Everything())
+	if err != nil {
+		return fmt.Errorf("failed to list pods at target namesapce %s: %w", nad.Namespace, err)
+	}
+
+	nadName := util.GetNADName(nad.Namespace, nad.Name)
+	var connectedPods []string
+	for _, pod := range pods {
+		networkByNamespaces, err := util.UnmarshalPodAnnotationAllNetworks(pod.Annotations)
```

Comment:
not sure I understand variable name `networkByNamespaces`, should it be called `podNetworks`?

Reply:
Done